internal/clients/nightfall/nightfall_internal_test.go

```diff
@@ -227,15 +231,41 @@ func TestCreateCommentsFromScanResp(t *testing.T) {
 					creditCard2Response,
 				},
 			},
-			haveTokenExclusionSet: tokenExclusionSet,
+			haveTokenExclusionList: tokenExclusionList,
 			want: []*diffreviewer.Comment{
 				createComment(creditCardResponse),
 			},
 			desc: "single credit card excluded",
 		},
+		{
```

Comment:
added test where exclusion list is `["4242-4242-4242-[0-9]{4}", "^127\\."]`, and detectors are `CREDIT_CARD` and `IP_ADDRESS`